services/movie_session.py

```diff
@@ -42,3 +42,9 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list:
+    session = MovieSession.objects.get(pk=movie_session_id)
+    all_seats = session.tickets.all()
+    return [{"row": place.row, "seat": place.seat} for place in all_seats]
```

Comment:
You could retrieve these values using Django ORM and `values` without iterating over objects in the list comprehension